wagtail/snippets/views/snippets.py

```diff
@@ -493,6 +493,11 @@ def get_queryset(self):
         )
 
 
+class InspectView(generic.InspectView):
+    view_name = "inspect"
+    any_permission_required = ["add", "change", "delete"]
```

Comment:
Would be nice to reuse Django's `view` permission here, but there will be some other areas that need to be changed, e.g. the permission check for the model index view that uses this function:

https://github.com/wagtail/wagtail/blob/853aad4305440336e2ce6b70769fdf827fe7922f/wagtail/snippets/permissions.py#L13-L19

I decided to leave this for now to avoid making the PR scope too big.